test/integration/osl_mongodb_parameters/controls/osl_mongodb_parameters.rb

```diff
@@ -0,0 +1,65 @@
+control 'osl_mongodb_paramters' do
+  describe yum.repo('mongodb-org') do
+    it { should exist }
+    its('baseurl') { should cmp %r{https\:\/\/repo\.mongodb\.org\/yum\/redhat\/[0-9]+\/mongodb-org\/4\.4\/.+} }
```

Comment:
This needs to be updated as it's failing now.

Reply:
Fixed!